drivers/wifi/nrfwifi/CMakeLists.txt

```diff
@@ -173,15 +173,15 @@ else()
   # Rebuild fw_load.c whenever the firmware binary changes
   # a bit crude way to do it, but it works
   add_custom_command(
-    OUTPUT ${CMAKE_SOURCE_DIR}/src/fw_load.c
-    COMMAND ${CMAKE_COMMAND} -E touch ${CMAKE_SOURCE_DIR}/src/fw_load.c
+    OUTPUT ${ZEPHYR_BASE}/drivers/wifi/nrfwifi/src/fw_load.c
+    COMMAND ${CMAKE_COMMAND} -E touch ${ZEPHYR_BASE}/drivers/wifi/nrfwifi/src/fw_load.c
```

Comment:
In no way should this code ever overwrite files in zephyr or touch files outside of the build folder

Reply:
As discussed, reverted the original commit and switched from incbin to native approach, thanks.